src/main/java/org/spburegistry/backend/entity/Student.java

```diff
@@ -0,0 +1,49 @@
+package org.spburegistry.backend.entity;
+
+import jakarta.annotation.Nullable;
+import jakarta.persistence.*;
+import lombok.*;
+import org.spburegistry.backend.enums.Degree;
+import org.spburegistry.backend.enums.Sex;
+
+import java.util.Set;
+
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="students")
+@Data
+public class Student extends BaseEntity {
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
```

Comment:
For enum you need add annotation: @Enumerated(EnumType.STRING)

Reply:
Done.